### PR TITLE
EDSC-1249 Added filter bands input and total bands vs selected counters.

### PR DESCRIPTION
**Pull request by mightynimble**

### CODE REVIEW

File: src/echoform.coffee [hunk 1]
```diff
@@ -49,6 +49,23 @@ class EchoForm
...
+          text = $('#bands-filter').val()
+          $('.jstree').jstree(true).search(text) if text.length > 1), 250
+
+      totalCount = $('.jstree').jstree(true).get_json?('#', flat: true).length
+      $('.jstree').on 'click', '.jstree-node', (e)->
+        checkedCount = $('.jstree').jstree(true).get_selected().length
+        $('#bands-count').text("#{checkedCount}/#{totalCount} bands selected")
+
+      checkedCount = $('.jstree').jstree(true).get_selected?().length
+      $('.jstree').prepend("<div id='bands-count'>#{checkedCount}/#{totalCount} bands selected</div>")
```
Comment (mreese84) [inline src/echoform.coffee]: Can you make this say "**n** of **n** bands selected"?

Comment (mreese84) [reply src/echoform.coffee]: Also, please make the numbers bold so they pop out a little more.

**Comment by mreese84** [comment]
@mightynimble so for forms that only have ONE top level node, is it possible to have that node auto expanded? It just looks really odd to only have a single node.

**Comment by mreese84** [comment]
@mightynimble I'm good with this, but it looks like tests aren't passing in Travis.

**Comment by mightynimble** [comment]
Thanks @mreese84 . Now I need one more approval from @jboukes or @macrouch on the code changes.

File: src/controls/extras/treeitem.coffee [hunk 2]
```diff
@@ -89,9 +89,10 @@ class TreeItem #extends Base
       data_jstree['selected'] = true
     unless @node_relevant()
       data_jstree['disabled'] = true
+#      data_jstree['selected'] = false
```
Comment (macrouch) [inline src/controls/extras/treeitem.coffee]: remove this line

**Review by macrouch — APPROVED** [review]

**Pull request merged**